manifests/params.pp

```diff
@@ -14,7 +14,9 @@
         }
       }
 
-      $package_names = ['katello', "${scl_prefix}rubygem-katello"]
+      $scl_package_names = ['rubygem-katello']
+      $scl_package_names_with_prefix = $scl_package_names.map |$pkg| { "${scl_prefix}${pkg}" }
```

Comment:
`map` is relatively new - 3.4 I think, yet we're testing with even 2.7 on travis.  stdlib includes a `prefix` function which might be a better option.